[CI] IndexLifecycleRestIT {yaml=ilm/40_explain_lifecycle/Test All Indexes Lifecycle Explain} failure

The test failed in one of my PR build:
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+pull-request-2/7623/console
but I can see some intermittent failures though the build is not available.
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob+fast+part2/1195/console

```
./gradlew ':x-pack:plugin:ilm:qa:rest:restTestRunner' \
--tests "org.elasticsearch.xpack.ilm.IndexLifecycleRestIT.test {yaml=ilm/40_explain_lifecycle/Test All Indexes Lifecycle Explain}" \
-Dtests.seed=147FD374FCB1376A \
-Dtests.security.manager=true \
-Dtests.locale=ja-JP-u-ca-japanese-x-lvariant-JP \
-Dtests.timezone=Indian/Cocos \
-Dcompiler.java=12 \
-Druntime.java=11
```

with error
```
org.elasticsearch.xpack.ilm.IndexLifecycleRestIT > test {yaml=ilm/40_explain_lifecycle/Test All Indexes Lifecycle Explain} FAILED
14:29:13     java.lang.AssertionError: Failure at [ilm/40_explain_lifecycle:218]: field [indices.index_with_policy_that_doesnt_exist.step_info.reason] is null
14:29:13 
14:29:13         Caused by:
14:29:13         java.lang.AssertionError: field [indices.index_with_policy_that_doesnt_exist.step_info.reason] is null
```

--- a/x-pack/plugin/ilm/qa/rest/src/test/resources/rest-api-spec/test/ilm/40_explain_lifecycle.yml
+++ b/x-pack/plugin/ilm/qa/rest/src/test/resources/rest-api-spec/test/ilm/40_explain_lifecycle.yml
@@ -162,10 +162,6 @@ teardown:
 
 ---
 "Test All Indexes Lifecycle Explain":
-  - skip:
-      reason: https://github.com/elastic/elasticsearch/issues/47275
-      version: "6.7.0 - "
-
   - do:
       ilm.explain_lifecycle:
         index: "*"
@@ -218,7 +214,6 @@ teardown:
 
   - match: { indices.index_with_policy_that_doesnt_exist.index: "index_with_policy_that_doesnt_exist" }
   - match: { indices.index_with_policy_that_doesnt_exist.policy: "a_policy_that_doesnt_exist" }
-  - match: { indices.index_with_policy_that_doesnt_exist.step_info.reason: "policy [a_policy_that_doesnt_exist] does not exist" }
   - is_true: indices.index_with_policy_that_doesnt_exist.managed
   - is_false: indices.index_with_policy_that_doesnt_exist.phase
   - is_false: indices.index_with_policy_that_doesnt_exist.action

